libxmp.map

```diff
@@ -70,3 +70,8 @@ XMP_4.5 {
     xmp_test_module_from_file;
     xmp_test_module_from_memory;
 } XMP_4.4;
+
+XMP_4.6 {
```

Comment:
Why 4.6? 4.5 isn't released yet.

Reply:
Because discussion elsewhere seemed to indicate a desire to freeze features for an impending 4.5 release.  Personally, I would be happy to include `xmp_add_channels` (and other, forthcoming additions) in version 4.5 but would require input from others before doing so.

Reply:
There was discussion for a 4.5 release but for whatever it was never actually tagged. I think shortly after that the fuzzing crash reports started coming in so there hasn't been much of an opportunity to freeze things since. I agree that this should be considered a 4.5 API function.